package.json

```
@@ -1,11 +1,16 @@
 {
   "name": "svelte-portal",
-  "version": "2.2.0",
+  "version": "2.2.1",
   "description": "Svelte component for rendering outside the DOM of parent component",
   "main": "src/main.cjs.js",
   "module": "src/main.es.js",
   "svelte": "src/Portal.svelte",
   "types": "src/Portal.svelte.d.ts",
+  "exports": {
+    ".": {
+      "types": "./src/Portal.svelte.d.ts",
+      "svelte": "src/main.es.js"
```

```suggestion
    ".": {
      "types": "./src/Portal.svelte.d.ts",
      "svelte": "src/main.es.js"
    }
```
Please add closing brace